pyro/contrib/gp/models/svgp.py

```diff
@@ -12,8 +12,28 @@
 
 
 class SparseVariationalGP(GPModel):
-    """
-    Sparse Variational Gaussian Process module.
+    r"""
+    Sparse Variational Gaussian Process model.
+
+    This model introduces an additional inducing-input parameter :math:`X_u` as in
+    :class:`~.SparseGPRegression` model and deals with both Gaussian and non-Gaussian
```

Comment:
Make the discussion of inducing points self-contained rather than referring to the contents of other docstrings